lib/fixture/locator/helpers.ts

```diff
@@ -42,13 +50,61 @@ const buildTestingLibraryScript = async ({config}: {config: Config}) => {
 const synchronousQueryNames = allQueryNames.filter(isNotFindQuery)
 
 const queriesFor = (pageOrLocator: Page | Locator) =>
```

Comment:
Would you mind exposing this publicly albeit undocumented/unofficial?

I have one test suite that [runs in serial mode with a shared `Page` object](https://playwright.dev/docs/test-retries#reuse-single-page-between-tests), and this test file therefore can't use the `{ page, queries }` fixtures for each test, and I have to set it up manually. (There might be a better way of doing this long-term, but for now this would help me.)

Reply:
(I have no issue with `within` being a fixture btw.)

Reply:
I was planning on exposing it in some form anyways to take the place of the legacy handle-based queries for the vanilla Playwright use case.